Update storage-tables-scale-targets.md to better describe max properties in a table

cesarwhite opened the pull request:
I provided feedback requesting clarification about the “Max number of properties in a table entity” listed at 252 in this section. I didn't know if the number accounted for the 3 system properties PartitionKey, RowKey and Timestamp for each entity. After some research, I found out the max number is 255 including the 3 properties mentioned above.  It would make more sense to indicate max of 255 and indicate/list the 3 properties.

@roygara provided a link to official MS documentation that corroborates this change

azurecla commented:
Hi __@snobu__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

PRmerger commented:
@snobu : Thanks for your contribution to the Azure documentation! The author, @ccompy, has been notified to review your proposed change.

Erikre commented:
This change can be merged.

cesarwhite committed:
Update storage-tables-scale-targets.md

Yesterday, I provided feedback requesting clarification about the “Max number of properties in a table entity” listed at 252 in this section. I didn't know if the number accounted for the 3 system properties PartitionKey, RowKey and Timestamp for each entity. After some research, I found out the max number is 255 including the 3 properties mentioned above.  It would make more sense to indicate max of 255 and indicate/list the 3 properties.
